README.md

```diff
@@ -5,10 +5,10 @@ This repository holds the [Constitution of the Iron Riders Robotics team](./Club
 ## Proposing a change using GitHub
 
 1. Make sure you have a GitHub account, register [here](https://github.com/join).
-2. Open up [Club Constitution.md](Club Constitution.md) and click the *Edit This File* button. This is the pencil icon in the top-right corner of the file preview.
+2. Open up [Club Constitution file](https://github.com/IronRiders/Constitution/blob/master/Club%20Constitution.md) and click the *Edit This File* button. This is the pencil icon in the top-right corner of the file preview.
```

Comment:
Relative link. `./Club Constitution.md` should work

Reply:
Yes but I wanted to make sure this worked regardless of fork/branch.